version2/aks-cluster.tf

```diff
@@ -0,0 +1,64 @@
+resource "random_pet" "prefix" {}
+
+provider "azurerm" {
+  subscription_id = var.subscription_id
+  client_id       = var.client_id
+  client_secret   = var.client_secret
+  tenant_id       = var.tenant_id
```

Comment:
Let's get rid of these. Provider auth should be done using environment variables or using the Azure CLI profile, this is so credentials aren't accidentally committed to version control